src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -1082,6 +1082,61 @@ The Split integration is only available for organizations on the Business and En
 Follow the instructions in the link below:  
 <https://help.split.io/hc/en-us/articles/360029879431>
 
+## Deployment
+
+These integrations will assist in automating your release and deployment process, making understanding your errors simpler.
+
+### Vercel
+
+Vercel is an all-in-one platform with Global CDN supporting static & JAMstack deployment and Serverless Functions. Connect your Sentry and Vercel projects to automatically upload source maps and notify Sentry of new releases being deployed.
+
+**Installation**
+
+1. Visit <https://vercel.com/integrations/sentry/add>
+
+2. Select your account and click "Add".
+
+    [{% asset vercel/vercel_install.png alt="Sentry modal showing Vercel installation." %}]({% asset vercel/vercel_install.png @path %})
+
+    **Note that the installation created an internal integration for the purpose of creating an auth token used in creating releases. If the Vercel internal integration is deleted, your Vercel integration will stop working!**
+
+    [{% asset vercel/vercel_internal_integration.png alt="Sentry modal showing Vercel internal integration." %}]({% asset vercel/vercel_internal_integration.png @path %})
+
+**Project Linking**
+
+1. Select a Sentry project and a Vercel project to link together.
+
+    [{% asset vercel/vercel_link_project.png alt="Sentry modal showing linking Sentry project to Vercel project." %}]({% asset vercel/vercel_link_project.png @path %})
+
+  * This step will generate environment variables in the selected Vercel project. Having these set means you can skip the [CLI Configuration step](https://github.com/getsentry/sentry-webpack-plugin#cli-configuration) in the Sentry Webpack Plugin setup. You can see these in Vercel in **Project Settings** > **General** > **Environment Variables**.
+
+
+    [{% asset vercel/vercel_env_vars.png alt="Sentry modal showing linking Sentry project to Vercel project." %}]({% asset vercel/vercel_env_vars.png @path %})
+
+    * `SENTRY_ORG` - the name of your Sentry organization
+    * `SENTRY_PROJECT` - the name of your linked Sentry project
+    * `SENTRY_AUTH_TOKEN` - the auth token from the Vercel Internal Integration that was created upon installation
+    * `NEXT_PUBLIC_SENTRY_DSN` - the linked Sentry project's DSN
+    * `VERCEL_GITHUB_COMMIT_SHA`, `VERCEL_GITLAB_COMMIT_SHA`, or `VERCEL_BITBUCKET_COMMIT_SHA` - your source code provider's commit sha, used for creating releases and associating commits
+
+**Usage**
+
+* If you have not already done so, [instrument your code with Sentry]({%- link _documentation/error-reporting/quickstart.md -%}).
+* Ensure you have [installed a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration).
```

Comment:
@ceorourke We also need to mention they need to add the repository of their project. But now I'm wondering if that could be automated during the setup.  hmmmm....